winrt/test.nativecomponent/winrt.test.nativecomponent.uap.vcxproj

```diff
@@ -34,6 +34,7 @@
       <AdditionalIncludeDirectories>..\Inc;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>_WINRT_DLL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <WarningLevel>Level4</WarningLevel>
+      <TreatWarningAsError>false</TreatWarningAsError>
```

Comment:
Why?   Policy is for Win2D to build without warnings...

Reply:
Without this the command line build failed with C4244 possible loss of data (in VS it builds without problem) [here](https://github.com/Microsoft/Win2D/blob/d0ef1aa3a93045b910fa67f31dddf1b41fdf70d1/winrt/test.nativecomponent/ShaderCompiler.cpp#L16)
My C/C++ skills are very basic so I did it this way. There are better ways for sure. Do you have a hint?

Reply:
Ah, I see what's happening there.  Looks like the newer compiler is smart enough to notice that the sloppy string conversion in that test code is indeed sloppy :-)

I haven't tested this, but something like the following should work (to replace the first std:;string constructor - repeat same pattern for each of the three strings):

    std::string asciiShader(shaderCode->Length(), ' ');
    std::transform(shaderCode->Begin(), shaderCode->End(), asciiShader.begin(), [](wchar_t c) { return static_cast<char>(c); });

Reply:
It works. Thanks